_posts/2016-06-30-state-in-scalable-architectures.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+name: State-in-scalable-architectures
+title: 'State in scalable architectures'
+date: 2016-06-30 00:20:00 +00:00
+author: Felipe Fernández
+image:
+    src: /assets/img/custom/blog/state.jpg
+    attribution:
+      text: Untitled by Daida Medina
+      href: https://www.instagram.com/p/8TdviFHo-g/?taken-by=thelastfreeusername
+tags:
+- state
+- scalability
+- architecture
+
+---
+
+Handling state is one of the cornerstone of software development. Most of business value derived from software relies on state. Depending on the level of abstraction, state, and the approach to handle it, looks like a complete different problem. In this post we'll see what is common to all of them and how modern scalability needs have pushed the industry to innovative and refreshing approaches.
```

Comment:
to innovative and refreshing approaches => to come up with innovative and refreshing approaches 

or

to come out with innovative and refreshing approaches.